src/ui/dlgPackageExporter.ui

```diff
@@ -90,36 +573,51 @@
    </item>
   </layout>
  </widget>
+ <tabstops>
+  <tabstop>packageList</tabstop>
+  <tabstop>packageName</tabstop>
+  <tabstop>openInfos</tabstop>
+  <tabstop>Author</tabstop>
+  <tabstop>addIcon</tabstop>
+  <tabstop>Title</tabstop>
+  <tabstop>Description</tabstop>
+  <tabstop>Dependencies</tabstop>
+  <tabstop>treeWidget</tabstop>
+  <tabstop>addedFiles</tabstop>
+  <tabstop>addFiles</tabstop>
+  <tabstop>packageLocation</tabstop>
+  <tabstop>filePath</tabstop>
+ </tabstops>
  <resources/>
  <connections>
   <connection>
    <sender>buttonBox</sender>
-   <signal>accepted()</signal>
```

Comment:
The changes on and below this line are just Git noise from the Qt designer just randomly swapping two signal-slot connections that are made from this file (rather than in the CPP class that has the slots) - I am not sure that all us developers are aware that such things which are equivalent to a `QObject::connect(...)` call in our C++ code can happen - but they can! :smiling_imp:

As it happens such connections also show up in one of the (lesser used) view tabs in the Qt Designer tool/plugin - IIRC this may or may not be the same one that shows buddy connections (click on a label for the focus to jump to the buddy) between labels and the field that they are buddy's with...